src/main/java/com/icthh/xm/uaa/config/UaaConfiguration.java

```diff
@@ -155,7 +163,20 @@ private TokenGranter tokenGranter(AuthorizationServerEndpointsConfigurer endpoin
             endpoints.getOAuth2RequestFactory(),
             tokenStore,
             authenticationManager);
+
+        IdpTokenGranter idpTokenGranter = new IdpTokenGranter(
```

Comment:
seems IdpTokenGranter should be returned as a @Bean

Reply:
After discussion we decided to leave it as it is